bitnami/grafana-operator/values.yaml

```diff
@@ -0,0 +1,211 @@
+## Default values for grafana-operator.
+## This is a YAML-formatted file.
+## Declare variables to be passed into your templates.
+
+## Global Docker image parameters
+## Please, note that this will override the image parameters, including dependencies, configured to use the global value
+## Current available global Docker image parameters: imageRegistry, imagePullSecrets
+##
+# global:
+#   imageRegistry: myRegistryName
+#   imagePullSecrets:
+#     - myRegistryKeySecretName
+
+## String to partially override metallb.fullname include (will maintain the release name)
+##
+# nameOverride:
+
+## String to fully override metallb.fullname template
+##
+# fullnameOverride:
+
+## Number of grafana-operator Pod replicas
+##
+replicaCount: 1
+
+## Set up update strategy for Grafana installation. Set to Recreate if you use persistent volume that cannot be mounted by more than one pods to makesure the pods is destroyed first.
+## ref: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy
+## Example:
+## updateStrategy:
+##  type: RollingUpdate
+##  rollingUpdate:
+##    maxSurge: 25%
+##    maxUnavailable: 25%
+##
+updateStrategy:
+  type: RollingUpdate
+
+image:
+  registry: quay.io
+  repository: integreatly/grafana-operator
+  tag: v3.6.0
+  ## Specify a imagePullPolicy
+  ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+  ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  ##
+  pullPolicy: IfNotPresent
+  ## Optionally specify an array of imagePullSecrets.
+  ## Secrets must be manually created in the namespace.
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ##
+  # pullSecrets:
+  #   - myRegistryKeySecretName
+
+args:
+  ## If one of these options is set a clusterRole and clusterRoleBinding is created to
+  ## ensure that the operator is able to get data from other namespaces.
+  scanAllNamespaces: false
+  scanNamespaces: []
+
+grafana:
+  image:
+    registry: docker.io
+    repository: bitnami/grafana
+    tag: 7.2.1
+    ## Optionally specify an array of imagePullSecrets.
+    ## Secrets must be manually created in the namespace.
+    ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+    ##
+    # pullSecrets:
+    #   - myRegistryKeySecretName
+
+grafanaPluginInit:
```

Comment:
In this case, we need to find an alternative since this image is not part of the bitnami catalog

Reply:
We adapted the functions to download the plugins so the `bitnami/grafana` container can act as plugin init container. This is the release that introduced the changes:

- `bitnami/grafana` [`7.3.3-debian-10-r1`](https://github.com/bitnami/bitnami-docker-grafana/commit/ca9bb328fe0dc672906e035c089d85f514e3298e)